src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -17,32 +16,61 @@ class SettingsLayout extends React.Component {
     routes: PropTypes.array,
   };
 
+  state = {
+    navVisible: false,
+  };
+
+  componentWillMount() {
```

Comment:
We should use `componentDidMount` as `WillMount` is deprecated

Reply:
Fixed in 619c649